R/plotColTile.R

```diff
@@ -99,19 +99,19 @@ setMethod("plotRowTile", signature = c("SummarizedExperiment"),
                               "row" = "rowData",
                               "column" = "colData")
     x_group <- data %>% 
-        group_by(.data$X) %>% 
+        group_by(X) %>% 
         summarise(group_n = n()) %>%
-        mutate(group_freq = .data$group_n/sum(.data$group_n),
-               x = cumsum(.data$group_freq),
-               xmin = c(0,.data$x[-length(.data$x)]))
+        mutate(group_freq = group_n/sum(group_n),
+               x = cumsum(group_freq),
+               xmin = c(0,x[-length(x)]))
```

Comment:
This should still work, so need to change?

Reply:
.data in tidyselect expressions was deprecated in 1.2.0